network/nxos/nxos_config.py

```diff
@@ -93,7 +93,7 @@
         without first checking if already configured.
     required: false
     default: false
-    choices: BOOLEANS
+    choices: [ "yes", "no" ]
```

Comment:
Agreement (from Ansible Team) is to use True/False. This (and other guidelines) will get documented.